src/priv-csrs.tex

```diff
@@ -124,6 +124,9 @@ \section{CSR Address Mapping Conventions}
 accessible to machine mode, whereas {\tt 0x7B0}--{\tt 0x7BF} are only visible
 to debug mode.  Implementations should raise illegal instruction exceptions on
 machine-mode access to the latter set of registers.
+In addition, supervisor-mode standard read-write CSRs {\tt 0x5A0}--{\tt 0x5AF}
+and hypervisor-mode standard read-write CSRs {\tt 0x6A0}--{\tt 0x6AF} are
+reserved for use by the debug system.
```

Comment:
I'm concerned that this makes it sound like the debug spec can allocate registers in these ranges at will, which is not what I hoped we would say.  I might be in the minority on this, but I think I'd prefer not to add this sentence.

Reply:
Works for me.  Change is made.